### PR TITLE
Work with InfluxDB logs

### LINKED ISSUE #3295: Can't find influxdb2 logging documentation page
_Describe the issue here._

##### Relevant URLs
influxdb1: https://docs.influxdata.com/influxdb/v1.4/administration/logs/

The link at the top of the v1 page (that says "InfluxDB v2.0 is the latest stable version.") does not go to a logging documentation page, but rather the influxdb2 base documentation: https://docs.influxdata.com/influxdb/v2.0/

The v1 documentation has an entire "Administration" section (of which "Logging" is a subsection) that I don't see in the v2 docs.

### PR DESCRIPTION
**Pull request by sanderson**
Closes #3295 

Adds documentation about managing logs in InfluxDB OSS 2.4.

- [x] Rebased/mergeable

### CODE REVIEW

**Comment by sanderson** [comment]
@lesam Adding you on this because I wasn't sure who from the Edge team should review this. Feel free to reassign.

**Review by jeffreyssmith2nd — APPROVED** [review]

File: content/influxdb/v2.4/admin/logs.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+influxd 1> /path/to/influxdb.log
+```
+
+{{% note %}}
+When logging to a file, InfluxDB uses the [logfmt](#logfmt) format.
+{{% /note %}}
+
+### Logs when running InfluxDB as a service
+
+If using a service manager to run InfluxDB, the service manager determines to location of logs.
```
Comment (sanderson) [inline content/influxdb/v2.4/admin/logs.md:43]:
```suggestion
If using a service manager to run InfluxDB, the service manager determines the location of logs.
```

File: content/influxdb/v2.4/admin/logs.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+
+For more information, see the [journald.conf documentation](https://www.freedesktop.org/software/systemd/man/journald.conf.html).
+
+{{% /tab-content %}}
+<!-------------------------------- END systemd -------------------------------->
+<!------------------------------- BEGIN sysvinit ------------------------------>
+{{% tab-content %}}
+
+When InfluxDB is run as a service, **stdout** is discarded by default (sent to `/dev/null`).
+To store write logs to a file:
```
Comment (sanderson) [inline content/influxdb/v2.4/admin/logs.md:68]:
```suggestion
To write logs to a file:
```

File: content/influxdb/v2.4/admin/logs.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+- [Configure your InfluxDB log location](#configure-your-influxdb-log-location)
+- [Configure your log level](#configure-your-log-level)
+- [Enable the Flux query log](#enable-the-flux-query-log)
+- [Use external tools to manage and process logs](#use-external-tools-to-manage-and-process-logs)
+- [Log formats](#log-formats)
+
+## Configure your InfluxDB log location
+
+By default, InfluxDB output all logs to **stdout**. To view InfluxDB logs,
```
Comment (noramullen1) [inline content/influxdb/v2.4/admin/logs.md:22]:
```suggestion
By default, InfluxDB outputs all logs to **stdout**. To view InfluxDB logs,
```

File: content/influxdb/v2.4/admin/logs.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+influxd 1> /path/to/influxdb.log
+```
+
+{{% note %}}
+When logging to a file, InfluxDB uses the [logfmt](#logfmt) format.
+{{% /note %}}
+
+### Logs when running InfluxDB as a service
+
+If using a service manager to run InfluxDB, the service manager determines the location of logs.
```
Comment (noramullen1) [inline content/influxdb/v2.4/admin/logs.md:43]:
```suggestion
If you use a service manager to run InfluxDB, the service manager determines the location of logs.
```

File: content/influxdb/v2.4/admin/logs.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+
+{{% /tab-content %}}
+<!-------------------------------- END sysvinit ------------------------------->
+{{< /tabs-wrapper >}}
+
+## Configure your log level
+
+Use the [`log-level` InfluxDB configuration option](/influxdb/v2.4/reference/config-options/#log-level)
+to specify the log levels the InfluxDB service outputs.
+The following log levels are supported:
```
Comment (noramullen1) [inline content/influxdb/v2.4/admin/logs.md:93]:
```suggestion
InfluxDB supports the following log levels:
```

File: content/influxdb/v2.4/admin/logs.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+_For information about configuring InfluxDB, see [InfluxDB configuration options](/influxdb/v2.4/reference/config-options/)._
+
+
+## Use external tools to manage and process logs
+
+Use the following popular tools to manage and process InfluxDB logs:
+
+### logrotate
+
+[logrotate](https://github.com/logrotate/logrotate) is designed to simplify the
```
Comment (noramullen1) [inline content/influxdb/v2.4/admin/logs.md:216]:
```suggestion
[logrotate](https://github.com/logrotate/logrotate) simplifies the
```

File: content/influxdb/v2.4/admin/logs.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+```sh
+2022-09-29T21:58:29.936355Z	info	Welcome to InfluxDB	{"log_id": "0dEoz3C0000", "version": "dev", "commit": "663d43d210", "build_date": "2022-09-29T21:58:29Z", "log_level": "info"}
+2022-09-29T21:58:29.977671Z	info	Resources opened	{"log_id": "0dEoz3C0000", "service": "bolt", "path": "/Users/exampleuser/.influxdbv2/influxd.bolt"}
+2022-09-29T21:58:29.977891Z	info	Resources opened	{"log_id": "0dEoz3C0000", "service": "sqlite", "path": "/Users/exampleuser/.influxdbv2/influxd.sqlite"}
+2022-09-29T21:58:30.059709Z	info	Checking InfluxDB metadata for prior version.	{"log_id": "0dEoz3C0000", "bolt_path": "/Users/exampleuser/.influxdbv2/influxd.bolt"}
+```
+
+### logfmt
+
+**When logging to a file**, logs are written in **logfmt**, a machine-readable
```
Comment (noramullen1) [inline content/influxdb/v2.4/admin/logs.md:264]:
```suggestion
**When logging to a file**, InfluxDB uses **logfmt**, a machine-readable
```

**Review by noramullen1 — APPROVED** [review]
Just a few small things

**Comment by sanderson** [comment]
Thanks for the review. I'm going to close this PR and use this branch as a base branch for a larger body of work for administrative tasks.

**Pull request closed without merging**